### PR TITLE
postman: Prevent null values

### PR DESCRIPTION
**Pull request by VitikaSoni**
## Overview
While working on error reporting, I observed that Postman's default values for attributes in body are set as empty strings. However, the users can potentially modify files, which could lead to exceptions in our addon.

## Checklist
- [x] Update help (N/A)
- [x] Update changelog (N/A)
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/postman/src/main/java/org/zaproxy/addon/postman/models/KeyValueData.java [hunk 1]
```diff
@@ -38,10 +38,10 @@ public KeyValueData(String key, String value) {
     }
 
     @JsonDeserialize(using = ObjectDeserializer.class)
-    private String key;
+    private String key = "";
```
Comment (thc202) [inline addOns/postman/src/main/java/org/zaproxy/addon/postman/models/KeyValueData.java:41]: This alone does not ensure that the string will be non-null, the user can explicitly set the field to null and it will end up being null here. For that it would have to be added something like `@JsonSetter(nulls = Nulls.SKIP)` too.

**Comment by thc202** [comment]
Would be good to add tests that verify that fields with null values do not lead to errors.

**Comment by VitikaSoni** [comment]
This is ready :)

**Review by kingthorin — APPROVED** [review]
LGTM

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!
